### PR TITLE
feat(query): http handler return session  state

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Fixes #issue

### CODE REVIEW

File: docs/doc/30-reference/00-api/00-rest.md [hunk 1]
```diff
@@ -262,22 +275,44 @@ set statement:
     "key": "max_threads",
     "value": "1",
     "is_global": false
+  },
+  "session_conf": {
```
[the remark this answers is not in the document]
Comment (youngsofun) [reply docs/doc/30-reference/00-api/00-rest.md:279]: ok to me

[the remark this answers is not in the document]
Comment (BohuTANG) [reply docs/doc/30-reference/00-api/00-rest.md:279]: session_state more better

**Review by BohuTANG — APPROVED** [review]
LGTM

**Comment by BohuTANG** [comment]
With this patch, I think we can do `set xx=<yy>` settings in databend cloud now, cc @Chasen-Zhang

**Comment by BohuTANG** [comment]
In this patch, we can not do cluster setting via 'set global <setting_key>=<setting_vslue>', any suggestions? @youngsofun

**Comment by youngsofun** [comment]
@BohuTANG   

1.  the  current client/session  do not need to distinguish  if it is global setting, the `session_state` in response do not need to  change
2.  client can 'set global <setting_key>=<setting_value>'  via SQL,  this setting should be recorded along with the warehouse/cluster,  its done by the query-server? not related to the handler?
3.  the only limit in the http handler  is  that it can not set global setting via "session_state" in the request json

**Pull request merged**